### ISSUE TITLE
Debian dh commands

### ISSUE THREAD

**Issue by haadiali72** [bluecherrydvr/bluecherry-apps#594]
I have created two different build environments with docker but the dh specific clean commands in the debian packaging give errors. I want to setup a development environment and access bluecherry on my local host/vm IP .In the first environment it gives the following error:
 "make[2]: Entering directory '/build/source/utils'
BCMK:6: *** recipe commences before first target.  Stop.
make[2]: Leaving directory '/build/source/utils'
make[1]: *** [Makefile:20: utils/clean] Error 2
make[1]: Leaving directory '/build/source'
dh_auto_clean: error: make -j1 clean returned exit code 2
make: *** [debian/rules:12: clean] Error 2"

The second error I face another dh clean error saying no configure.guess found in /misc/libconfig/libconfig/aux-build eventho i can see the file.

**Comment by haadiali72** [comment]
@curtishall This script gives the same errors. This script is where issue originates essentially

**Comment by haadiali72** [comment]
@curtishall yes i have in both of them. The second error is strange. The real path is misc/libconfig/ aux-build/config.guess while it is reading it as misc/libconfig/libconfig/aux-build

**Comment by haadiali72** [comment]
I want to access it on localhost/vm IP for customisation

**Comment by curtishall** [comment]
Okay, since you are using docker you should be able to follow these scripts:

https://github.com/bluecherrydvr/bluecherry-apps/tree/master/scripts/build-scripts-docker

**Comment by haadiali72** [comment]
@curtishall the docker scripts give the same issue when building. Is the issue rectified?

**Comment by haadiali72** [comment]
I have tried building with both and last I checked it was giving the same error with docker
